spec/services/create_diagnosis/find_relevant_expert_subjects_spec.rb

```diff
@@ -2,6 +2,168 @@
 
 require 'rails_helper'
 describe CreateDiagnosis::FindRelevantExpertSubjects do
+  describe 'apply_match_filters' do
+    subject{ described_class.new(need).apply_match_filters(ExpertSubject.all) }
+
+    let!(:es_temoin) { create :expert_subject }
+
+    context 'min_years_of_existence' do
+      let(:diagnosis) { create :diagnosis, company: company }
+      let(:need) { create :need, diagnosis: diagnosis }
+      let!(:es_01) { create :expert_subject }
+      let(:match_filter_01) { create :match_filter, min_years_of_existence: 3 }
+
+      before { es_01.expert.antenne.match_filters << match_filter_01 }
+
+      context 'young company' do
+        let(:company) { create :company, date_de_creation: 2.years.ago }
+
+        it do
```

Comment:
je chipote mais on peut peut être mettre les `it` sur une seule ligne pour réduire le nombre de lignes du fichier, il y en a pas mal

Reply:
yup, t'as raison :) 
Je l'ai fait sur plusieurs lignes pour des motifs de debug, plus besoin maintenant